experimental/provider_ppucd/tests/props_test.rs

```diff
@@ -29,27 +29,13 @@ fn test_enum_props_getters() {
     let ppucd_property_file_str = std::fs::read_to_string(ppucd_property_files_root_path).unwrap();
     let ppucd_provider: PpucdDataProvider = PpucdDataProvider::new(&ppucd_property_file_str);
 
-    // lb=LF
-    let lb_lf_uniset: UnicodeSet =
-        icu_uniset::props::get_line_break_val_set(&ppucd_provider, LineBreak::LineFeed).unwrap();
-    let exp_lb_lf_uniset: UnicodeSet = UnicodeSet::from_inversion_list(vec![10, 11]).unwrap();
-    assert_eq!(lb_lf_uniset, exp_lb_lf_uniset);
-}
-
-#[test]
-fn test_enum_props_getters_truncated_resc_key_subcategory() {
-    let ppucd_property_files_root_path = "tests/testdata/ppucd-inpc-topbotleft-test.txt";
-    let ppucd_property_file_str = std::fs::read_to_string(ppucd_property_files_root_path).unwrap();
-    let ppucd_provider: PpucdDataProvider = PpucdDataProvider::new(&ppucd_property_file_str);
-
-    // InPC=Top_And_Bottom_And_Left
-    let inpc_topbotleft_uniset: UnicodeSet =
-        icu_uniset::props::get_indic_positional_category_val_set(
-            &ppucd_provider,
-            IndicPositionalCategory::TopAndBottomAndLeft,
-        )
-        .unwrap();
-    let exp_inpc_topbotleft_uniset: UnicodeSet =
-        UnicodeSet::from_inversion_list(vec![4156, 4157, 71454, 71455]).unwrap();
-    assert_eq!(inpc_topbotleft_uniset, exp_inpc_topbotleft_uniset);
+    // gc=Zs
+    let gc_zs_uniset: UnicodeSet = icu_uniset::props::get_general_category_val_set(
+        &ppucd_provider,
+        GeneralCategory::SpaceSeparator,
+    )
+    .unwrap();
+    let exp_gc_zs_uniset: UnicodeSet =
+        UnicodeSet::from_inversion_list(vec![32, 33, 160, 161]).unwrap();
```

Comment:
Please use hex constants for code points.

Reply:
Done.